src/hip/batched_transpose_sol.cpp

```diff
@@ -379,4 +379,32 @@ size_t BatchedTransposeSolution::GetSize() const
     return miopen::GetTypeSize(data_type) * batch * height * width;
 }
 
+TransposeSolutionWorkspaceBufTraits::TransposeSolutionWorkspaceBufTraits(std::initializer_list<size_t> v_size_, 
```

Comment:
Let's make it available for general use. Move this to buffer_info.hpp/cpp and rename to `MultiBufferWorkspaceTraits`. What do you think?

Reply:
Agreed. I'll move it.

Reply:
moved and renamed.